uPortal-health/src/main/java/org/apereo/portal/health/DatabaseHealthChecker.java

```diff
@@ -0,0 +1,99 @@
+package org.apereo.portal.health;
+
+import com.sun.istack.NotNull;
```

Comment:
Seems like there is a better option then the `com.sun.istack` annotation.  I did some research, but there was no clear winner.  Once uP gets to Spring 5, there are Spring annotations that would suffice - https://www.baeldung.com/spring-null-safety-annotations

Reply:
Yep